apps/asap-server/src/routes/events.route.ts

```diff
@@ -25,7 +26,7 @@ export const eventRouteFactory = (eventController: EventController): Router => {
       const { params } = req;
       const { eventId } = framework.validate('parameters', params, paramSchema);
 
-      const result = await eventController.fetchById(eventId);
+      const result = await eventController.fetchById(eventId!);
```

Comment:
instead of non-null assertion why dont you use something like this in your route definition

```typescript
eventRoutes.get<{ eventId: string}>('/events/:eventId', async (req, res) => {
```
It's still technically just asserting but at least you have the param definition right next to it so i think it's legit